packages/OreIdProfile/.gitignore

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
It looks like this repo uses the root .gitignore instead of a package specific .gitignore. Do we need to break this convention?

Reply:
This project has lerna, but lerna doesn't work...

So from what I understand, it's a repository that stores all the "react libs". I personally don't like this pattern I prefer each package to have its own repository, but this is how it works.

That said, I believe .gitignore should be inside every package. Same as aikon-library